### PR TITLE
Adds rake task for getting last incremented solr collection

### PR DESCRIPTION
**Pull request by cdmo**

### CODE REVIEW

File: lib/psulib_traject/solr_manager.rb [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    def connection
+      @connection ||= Faraday.new(ConfigSettings.solr.url) do |faraday|
+        faraday.request :multipart
+        faraday.adapter :net_http
+      end
+    end
+  end
+
+  # Lightweight Solr collection abstraction
+  class SolrCollection
```
Comment (banukutlu) [inline lib/psulib_traject/solr_manager.rb:44]: nice 👍 maybe we implement this abstraction to BlackCat solrmanager?

**Review by banukutlu — APPROVED** [review]
👍

File: lib/psulib_traject/solr_manager.rb (hunk 1 above)
Comment (cdmo) [reply lib/psulib_traject/solr_manager.rb:44]: Yeah, I thought this was an improvement too, wanted to hear your thoughts. I think I wish SolrManager was just a separate plugin we could use anywhere, if time were infinite that's what I'd prefer

Comment (banukutlu) [reply lib/psulib_traject/solr_manager.rb:44]: yes! how nice that would be 💯

**Pull request merged**